src/aap_eda/settings/default.py

```diff
@@ -391,7 +393,7 @@ def _rq_common_parameters():
             "HOST": REDIS_HOST,
             "PORT": REDIS_PORT,
         }
-        if REDIS_CLIENT_CERT_PATH:
+        if REDIS_CLIENT_CERT_PATH or REDIS_TLS:
```

Comment:
```suggestion
        if (REDIS_CLIENT_CERT_PATH and (REDIS_TLS is None)) or REDIS_TLS:
```
This provides backwards compatibility where TLS is not specified and  respects the setting of TLS, if specified.  That is, if TLS is specified False it supersedes the previous implicit setting.

Reply:
@jshimkus-rh Since this is a new field and its in the same file settings/default.py why does it have to be None, are you thinking about the settings.yml file that the customer might have instead of the python file. Even in that case it would be inheriting the default values defined in the settings.py first before it overrides values from the settings.yml.

Reply:
@mkanoor As currently coded if REDIS_TLS is actually specified (via settings/env vars) as False the code would ignore it and enable TLS regardless if REDIS_CLIENT_CERT_PATH were specified.
If it is set we must assume it was purposefully done so and respect the value.

Reply:
The logic behind setting REDIS_TLS to False by default is to align with the current behavior.  That behavior is to not use TLS unless we activate it by setting REDIS_CLIENT_CERT_PATH.  Therefore, REDIS_TLS should be set to False by default.  The only reason to deviate from that is if there is another way to activate TLS for EDA.  I am not the most familiar with the code base but that didn't seem like a possibility when I reviewed the code.

Reply:
@dhageman We should respect (backwards compatibility) the implicit setting of TLS = True if TLS is not separately specified.  
With this change we now have to ability to have TLS enablement specifically chosen by the user and we must assume that, for whatever purpose they may have, if it is provided and False that is what the user wants.

Reply:
@jshimkus-rh - Okay, the so the condition you want to address if someone explicitly sets the REDIS_CLIENT_CERT_PATH to <something> and REDIS_TLS to False that you want to NOT enable TLS?

Reply:
@dhageman 
> Okay, the so the condition you want to address if someone explicitly sets the REDIS_CLIENT_CERT_PATH to "something" and REDIS_TLS to False that you want to NOT enable TLS?

That is correct.

Reply:
I really think this highlights a bigger issue.  There should only be one option to enable TLS so it is explicitly clear what is occurring.  The use of enabling TLS via REDIS_CLIENT_CERT_PATH should be deprecated and eventually removed.  I wonder if the better approach is to document the behavior today and then move toward making REDIS_TLS the only option to enable TLS going forward.  Thoughts?

Reply:
@dhageman That is an option.  I didn't propose it in order to avoid the delay it would  add to getting this change in asap.  As long as it doesn't _require_ any changes to existing installers, etc. we can get this in right away and pursue anything further later.

Reply:
I have updated the pull request and put in a comment on there on the ideal path forward.  We should get this in the next round of deprecation notices.  This will work in the interim.